client/src/lib.rs

```diff
@@ -518,6 +523,11 @@ pub trait ServerGatewayApis {
     fn get_options(&self) -> &ServerGatewayOptions;
 }
 
+impl WorkflowServiceClientTrait for ServerGateway {
+    // TODO: Will implement 1:1 generated client methods. Boilerplate bonanza
```

Comment:
I'd be ok if you wanted to just get rid of this trait and ask everyone to use the actual type.

Reply:
Problem is the retryer needs to be able to wrap the raw thing too (Otherwise the only benefit we get is connection sharing, and not also retry logic, at which point... why bother). 

So, since the retryer should be transparently usable in place of a non-retryer thing, we need some trait for it, and hence we need that 1:1 trait.

So, no way around it. I'll just apply a little macro magic.

Reply:
So I reconsidered here after back and forth and decided to go with a little helper macro for calling the raw functions. See here: https://github.com/temporalio/sdk-core/pull/265/files#diff-3d28ff5c2d21a40cf191650a5a2043817ee022f49bbd012fb3c6526add336ce7R159